libs/hdf-converters/src/fortify-mapper.ts

```diff
@@ -181,13 +180,36 @@ export class FortifyMapper extends BaseConverter {
         ],
         sha256: ''
       }
-    ]
+    ],
+    passthrough: {
+      transformer: (data: Record<string, unknown>): Record<string, unknown> => {
+        let auxData = _.get(data, 'FVDL');
+        if (auxData instanceof Object) {
```

Comment:
use lodash's check for if something is an object: https://masteringjs.io/tutorials/fundamentals/instanceof